### PR TITLE
feat: no floating promises in extension

### PR DESCRIPTION
**Pull request by mateuszjasiuk**
Added no floating promises.
In the effects we have to be explicit with void prefix, but I don't think this is a huge deal :)

### CODE REVIEW

File: apps/extension/src/background/approvals/service.test.ts [hunk 1]
```diff
@@ -94,7 +94,11 @@ describe("approvals service", () => {
     it("should throw an error when popupTabId is not present", async () => {
       jest.spyOn(service as any, "getPopupTabId").mockResolvedValue(undefined);
 
-      expect(service.approveSignature("signer", "data")).rejects.toBeDefined();
+      try {
```
Comment (emccorson) [inline apps/extension/src/background/approvals/service.test.ts:97]: I think in these cases you could also just await or return:

```
await expect(service.approveSignature("signer", "data")).rejects.toBeDefined();
```

**Review by emccorson — APPROVED** [review]
Looks good to me!

File: apps/extension/src/background/approvals/service.test.ts (hunk 1 above)
Comment (mateuszjasiuk) [reply apps/extension/src/background/approvals/service.test.ts:97]: Yeah I've noticed that a bit later :D

Comment (mateuszjasiuk) [reply apps/extension/src/background/approvals/service.test.ts:97]: I fixed that in another PR :)

**Pull request merged**